.circleci/config.yml

```diff
@@ -29,7 +33,7 @@ jobs:
       - run:
           name: List versions
           command: |
-            echo "npm: $(npm --version)"
+            echo "yarn: $(yarn --version)"
             echo "node: $(node --version)"
       - checkout
```

Comment:
In Falcon, we also keep a cache for `.git`.
See https://github.com/plotly/falcon-sql-client/blob/3ebfd4d20a14973f6023990fabfff0cc285e0b21/circle.yml#L12-L21

Reply:
Okay, @n-riesco, how about now?: https://circleci.com/gh/plotly/image-exporter/1175#config/containers/0